wire-ios/Wire-iOS/Sources/UserInterface/Settings/Debug Report/SettingsDebugReportViewController.swift

```diff
@@ -0,0 +1,168 @@
+            infoLabel.topAnchor.constraint(equalTo: view.safeTopAnchor, constant: LayoutConstants.padding),
+            infoLabel.leadingAnchor.constraint(equalTo: view.leadingAnchor, constant: LayoutConstants.padding),
+            infoLabel.trailingAnchor.constraint(equalTo: view.trailingAnchor, constant: -LayoutConstants.padding),
+
+            shareReportButton.bottomAnchor.constraint(equalTo: view.safeBottomAnchor, constant: -LayoutConstants.safeBottomPadding),
+            shareReportButton.leadingAnchor.constraint(equalTo: view.leadingAnchor, constant: LayoutConstants.padding),
+            shareReportButton.trailingAnchor.constraint(equalTo: view.trailingAnchor, constant: -LayoutConstants.padding),
+            shareReportButton.heightAnchor.constraint(greaterThanOrEqualToConstant: LayoutConstants.buttonHeight),
+
+            sendReportButton.bottomAnchor.constraint(equalTo: shareReportButton.topAnchor, constant: -LayoutConstants.spacing),
+            sendReportButton.leadingAnchor.constraint(equalTo: view.leadingAnchor, constant: LayoutConstants.padding),
+            sendReportButton.trailingAnchor.constraint(equalTo: view.trailingAnchor, constant: -LayoutConstants.padding),
+            sendReportButton.heightAnchor.constraint(greaterThanOrEqualToConstant: LayoutConstants.buttonHeight)
+        ])
+    }
+
+    private func setupNavigationBarTitle() {
+        // Set the navigation item title
+        navigationItem.title = Strings.TechnicalReportSection.title.capitalized
+
+        // Set the navigation bar title color and font
+        let titleTextAttributes: [NSAttributedString.Key: Any] = [
+            .foregroundColor: SemanticColors.Label.textDefault,
+            .font: UIFont.font(for: .h3)
+        ]
+        navigationController?.navigationBar.titleTextAttributes = titleTextAttributes
+
+        // Ensure title supports LargeContentViewer
+        navigationItem.titleView?.isAccessibilityElement = true
+        navigationItem.titleView?.accessibilityTraits = .header
+        navigationItem.titleView?.accessibilityLabel = navigationItem.title
+
+        if let titleView = navigationItem.titleView {
+            titleView.showsLargeContentViewer = true
+            titleView.largeContentTitle = navigationItem.title
+        }
+    }
+
+    // MARK: - Actions
+
+    @objc private func didTapSendReport() {
+        viewModel.sendReport()
+    }
+
+    @objc private func didTapShareReport() {
+        viewModel.shareReport()
+    }
+
+    // MARK: - Helpers
+
+    private func createButton(title: String, action: UIAction) -> UIButton {
+        let button = ZMButton(
+            style: .secondaryTextButtonStyle,
+            cornerRadius: 16,
+            fontSpec: .buttonBigSemibold
+        )
+        button.setTitle(title, for: .normal)
+        button.addAction(action, for: .touchUpInside)
+        button.translatesAutoresizingMaskIntoConstraints = false
+        return button
+    }
+}
```

Comment:
I would recommend replacing this init with one using the new WireTextStyle.

Reply:
`ZMButton` doesn't have this kind of init, and implementing it is out the scope of this PR